### PR TITLE
Create Modules-Index.md

### PR DESCRIPTION
**Pull request by Svitlana007**

### CODE REVIEW

File: Modules-Index.md [hunk 1]
```diff
@@ -0,0 +1,91 @@
+## Hi there 👋
+
+### Documentation: [LLazyEmail Documentation](https://llazyemail.github.io/documentation/)
+
+<!--
```
Comment (atherdon) [inline Modules-Index.md:5]: We can remove hidden Block, its not needed here

File: Modules-Index.md [hunk 2]
```diff
@@ -0,0 +1,91 @@
...
+- [React-email-typography-components](https://github.com/LLazyEmail/react-email-typography-components)
+- [Nextjs-preview](https://github.com/LLazyEmail/nextjs-preview)
+- [React-email-miscellaneous-components](https://github.com/LLazyEmail/react-email-miscellaneous-components)
+- [React-sendgrid-form](https://github.com/LLazyEmail/react-sendgrid-form)
+- [React-email-template](https://github.com/LLazyEmail/react-email-template)
+- [React-email-table](https://github.com/LLazyEmail/react-email-table)
+- [Email-template-react](https://github.com/LLazyEmail/email-template-react)
+- [React-email-letter-components](https://github.com/LLazyEmail/react-email-letter-components)
+
+### Cool projects to see
```
Comment (atherdon) [inline Modules-Index.md:81]: We can remove this section

**Review by atherdon — APPROVED** [review]

File: Modules-Index.md (hunk 1 above)
Comment (Svitlana007) [reply Modules-Index.md:5]: https://github.com/LLazyEmail/documentation/pull/163

File: Modules-Index.md (hunk 2 above)
Comment (Svitlana007) [reply Modules-Index.md:81]: https://github.com/LLazyEmail/documentation/pull/163

**Pull request merged**